extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -125,6 +128,32 @@ class AmpStickyAd extends AMP.BaseElement {
       });
     }
   }
+
+  /**
+   * The function that add a close button to sticky ad
+   * @private
+   */
+  addCloseButton_() {
+    const closeButton = this.getWin().document.createElement('div');
+    closeButton.classList.add('-amp-sticky-ad-close-button');
+    closeButton.setAttribute('aria-label','Close');
```

Comment:
What I mean this something like

`closeButton.setAttribute('aria-label', this.element.getAttribute('data-close-button-aria-label') || 'Close');`